src/components/OasisMarketHistory.jsx

```diff
@@ -1,18 +1,53 @@
 import React, { PureComponent } from 'react';
 import { PropTypes } from 'prop-types';
-// import ImmutablePropTypes from 'react-immutable-proptypes';
+import moment from 'moment';
+
+import OasisWidgetFrame from '../containers/OasisWidgetFrame';
+import { OasisTable } from './OasisTable';
+import { tradeType, price, formatTradeType, formatPrice, getBaseAndQuotePrice } from '../utils/tokens/pair';
+import { orderBytTimestamp, SORT_DESC } from '../utils/sort';
 
-import styles from './OasisMarketHistory.scss';
+// import ImmutablePropTypes from 'react-immutable-proptypes';
 
-const propTypes = PropTypes && {};
+const propTypes = PropTypes && {
+  activeTradingPair: PropTypes.object.isRequired,
+  trades: PropTypes.array.isRequired,
+};
 const defaultProps = {};
 
+const colDefinition = (baseToken, quoteToken) => {
```

Comment:
it's an array right , so colsDefinition might be more appropriate

Reply:
You suggested this name before.